scripts/tokenomics-emulation/nominators.ts

```diff
@@ -0,0 +1,161 @@
+import { KeyringPair } from "@polkadot/keyring/types";
+import Network from "./network";
+
+class Nominator {
+  private stashBalance: string;
+  private controllerBalance: string;
+
+  constructor(
+    private readonly network: Network,
+    private readonly decimals: number
+  ) {}
+
+  /**
+   * wait till node sync
+   * @param waitSeconds wait seconds
+   */
+  public async start(waitSeconds: string) {
+    console.log("Check if syncing...");
+    await this.callWithRetry(
+      this.isSyncing.bind(this),
+      {
+        maxDepth: 100,
+      },
+      0,
+      waitSeconds
+    );
+    console.log("Sync is complete!");
+  }
+
+  /**
+   * Load stash and controller accounts.
+   */
+  public async accountsBalance(
```

Comment:
What are you doing with accounts? 
Reread our guidelines (naming sections).

Reply:
Removed and passing the values as params